pkg/operator/apis/monitoring/v1/operator_types.go

```diff
@@ -88,6 +88,13 @@ type CollectionSpec struct {
 	KubeletScraping *KubeletScraping `json:"kubeletScraping,omitempty"`
 	// Compression enables compression of metrics collection data
 	Compression CompressionType `json:"compression,omitempty"`
+	// Export enables exporting metrics to an endpoint.
+	Export ExportSpec `json:"export,omitempty"`
+}
+
+type ExportSpec struct {
```

Comment:
Nice, this unblocks simple remote write cases, but I wonder if we could clear up some consistency/functionality questions before moving forward: 

1. Should add type of export to be explicit if this is PRW, Otel or GCM API? If not (we can add type field later on, once use case appears) we could claim it PRW for now, but let's then specify that in commentary. Also let's mention this is ON TOP of automatic Cloud Monitoring export.
2. To be future proof, I wonder if Exports []ExportSpec wouldn't be more appropriate.
3. What's the relation of ExportSpec.URL to `Filter` `ExternalLabel` `Credentials` `Compression` etc? Filter would be nice to have (some translation to relabelling would be necessary), but wonder if `exports` should use those collection fields, or have their own? For filtering, I would say per export make more sense from user perspective, but open to anything. For external label e.g. we can't really do that per export as it's done per collector. Ideally we somehow separate what's per export, what's "global/cluster wide".

Reply:
PTAL